From c93185a837f86b0288225b7f1dcf3a8862ec150a Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:40:03 -0400
Subject: [ethernet] i40e: remove unnecessary break after goto

Message-id: <1409658042-22689-243-git-send-email-sassmann@redhat.com>
Patchwork-id: 90445
O-Subject: [RHEL7.1 PATCH 242/281] i40e: remove unnecessary break after goto
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Fabian Frederick <fabf@skynet.be>

Signed-off-by: Fabian Frederick <fabf@skynet.be>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 3ec9fa753543d17d9a21c85bd2bb6f0d02f7f02f)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_lan_hmc.c b/drivers/net/ethernet/intel/i40e/i40e_lan_hmc.c
index 0d74b46..4627588 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_lan_hmc.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_lan_hmc.c
@@ -417,7 +417,6 @@ static i40e_status i40e_create_lan_hmc_object(struct i40e_hw *hw,
    default:
     ret_code = I40E_ERR_INVALID_SD_TYPE;
     goto exit;
-    break;
    }
   }
  }
@@ -502,7 +501,6 @@ try_type_paged:
   hw_dbg(hw, "i40e_configure_lan_hmc: Unknown SD type: %d\n",
      ret_code);
   goto configure_lan_hmc_out;
-  break;
  }
 
  /* Configure and program the FPM registers so objects can be created */
-- 
1.7.1